sbws/util/config.py

```diff
@@ -21,60 +24,67 @@
 log = logging.getLogger(__name__)
 
 
-def _read_config_file(conf, fname):
-    assert os.path.isfile(fname)
+def _expand_path(path):
+    """Expand path string containing shell variables and ~ constructions
+    into their values.
+    """
+    return os.path.expanduser(os.path.expandvars(path))
+
+
+def _extend_config(conf, fname):
+    """Extend ConfigParser from file configuration."""
     log.debug('Reading config file %s', fname)
     with open(fname, 'rt') as fd:
         conf.read_file(fd, source=fname)
     return conf
 
 
 def _get_default_config():
-    conf = ConfigParser(interpolation=ExtendedInterpolation())
-    fname = os.path.join(PKG_DIR, 'config.default.ini')
-    assert os.path.isfile(fname)
-    conf = _read_config_file(conf, fname)
-    return conf
+    """Return ConfigParser with default configuration."""
+    conf = ConfigParser(interpolation=ExtendedInterpolation(),
+                        converters={'path': _expand_path})
+    return _extend_config(conf, DEFAULT_CONFIG_PATH)
 
 
 def _get_user_config(args, conf=None):
+    """Get user configuration.
+    If a custom path is passed, search for user configuration there.
+    In any case, create a minimal user configuration in user configuration path
+    , in case it needs to be overwritten.
+    """
```

Comment:
This function doesn't currently create a file. I think that just means the comment needs to be updated.